src/main/java/com/connexta/search/query/QueryManagerImpl.java

```diff
@@ -45,6 +45,34 @@ public QueryManagerImpl(
     this.dataStore = dataStore;
   }
 
+  /**
```

Comment:
I had IntelliJ's option to reorder code enabled and this method got moved. I've disabled that in IntelliJ so that it won't happen again.